play-frontend-hmrc-play-30/src/test/scala/uk/gov/hmrc/hmrcfrontend/views/helpers/hmrcInternalHeadSpec.scala

```diff
@@ -56,15 +56,13 @@ class hmrcInternalHeadSpec
       scripts.first.attr("nonce") should be("a-nonce")
     }
 
-    "include a nonce in the IE9+ link tag if supplied" in {
+    "include a nonce in the link tag if supplied" in {
       val hmrcInternalHead = app.injector.instanceOf[HmrcInternalHead]
       hmrcfrontend.RoutesPrefix.setPrefix("/some-service/hmrc-frontend")
       val content          = contentAsString(hmrcInternalHead()(requestWithNonce, implicitly))
 
       content should include regex
-        """<!--\[if gt IE 8\]><!-->
-          |<link href="/some-service/hmrc-frontend/assets/hmrc-frontend-\d+.\d+.\d+.min.css" nonce="a-nonce" media="all" rel="stylesheet" type="text/css" />
-          |<!--<!\[endif\]-->""".stripMargin.r
+        """<link href="/some-service/hmrc-frontend/assets/hmrc-frontend-\d+.\d+.\d+.min.css" nonce="a-nonce" media="all" rel="stylesheet" type="text/css" />""".r
```

Comment:
total aside, but I think maybe we don't need a nonce on css being included by links but just ones included by <style> tags inline - I'm not certain - just sharing a thought I've had in back of my mind since spending more time iwth CSP stuff

Reply:
Looks like `style-src` [applies to all CSS](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/style-src), whether from a `<style>` or a `<link>` tag

Reply:
ah, cool - so we don't have style src in the default bootstrap csp but we should keep adding it in library in case some teams do 🙌